package.json

```
@@ -14,6 +14,10 @@
       "import": "./dist/index.js"
     }
   },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:vitejs/release-scripts.git"
```

We are using this format in other projects in the org, so maybe good to stick with it
```suggestion
    "url": "git+https://github.com/vitejs/release-scripts.git"
```